common/src/main/java/io/apicurio/registry/rest/AdminResource.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2020 Red Hat
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.apicurio.registry.rest;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+
+/**
+ * JAX-RS interface for the admin API of Apicurio Registry.
+ */
+@Path("admin")
+public interface AdminResource {
+
+    /**
+     * Returns the configured log level for the provided loggerName. If level is provided it configures the logger with the given log level.
```

Comment:
@Apicurio/developers before I update the open api spec, have you checked this? do you like this api design? I'm not 100% sure, here I'm using an optional query param to make the operation return or modify the logger. wdyt, should I split this to a GET and POST operations?

Reply:
Yeah, definitely make this two operations.  `PUT` or `POST`, I'm not sure which is better in this case.  But `GET` should never make a change.

Reply:
+1 to the two operations and to avoid a `GET` to make changes.